build/debs/BUILD

```diff
@@ -86,6 +87,12 @@ pkg_tar(
     deps = ["@kubernetes_cni//file"],
 )
 
+pkg_tar(
+    name = "cri_tools-data",
```

Comment:
`cri_tools_data`?

Reply:
the -data is required because i'm using the k8s_deb rule:

https://github.com/kubernetes/repo-infra/blob/master/defs/deb.bzl#L26